site/src/nav/DownloadButton.jsx

```diff
@@ -81,25 +86,74 @@ function DownloadButton() {
     setLoading(false);
   };
 
+  const handleEnterButton = () => {
+    if (zoom < ZOOM_BOUND) {
+      setShowFloater(true);
+    }
+  };
+
+  const handleLeaveButton = () => {
+    if (zoom < ZOOM_BOUND) {
+      setShowFloater(false);
+    }
+  };
+
   return (
-    <div className="button--download">
-      <button
-        className={`button button--primary ${loading ? "disabled" : ""}`}
-        onClick={handleDownloadClick}
-      >
-        <div className="wrapper">
-          <div className="download-icon">
-            {!loading ? (
-              <img className={"dl-img"} src={downloadIcon} />
-            ) : (
-              <RefreshIcon />
-            )}
+    <div>
+      <Floater
+        styles={{
+          container: {
+            borderRadius: "10px",
+            padding: "10px",
+            color: mode === "theme-dark" ? "white" : "black",
+            fontSize: ".7rem",
+            background:
+              mode === "theme-dark"
+                ? "var(--ifm-navbar-background-color)"
+                : "var(--ifm-color-secondary-light)",
+          },
+          arrow: {
+            color:
+              mode === "theme-dark"
+                ? "var(--ifm-navbar-background-color)"
+                : "var(--ifm-color-secondary-light)",
+          },
+        }}
+        content={
+          <div>
+            The download button is disabled at zoom levels below {ZOOM_BOUND}.
+            This is done to prevent downloading large amounts of data. To
+            reenable the button, zoom further in.
           </div>
-          <div className="download-text">
-            {loading ? "Downloading..." : "Download Visible"}
+        }
+        open={showFloater}
```

Comment:
Floater is "controlled", and only is shown when use is hovering over the 'button--download' div element.